docs/feature/proxy-support.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Connecting a Minestom server to a velocity proxy
+---
+
+# Proxy Support
+
+Minestom by default supports [Velocity](https://papermc.io/software/velocity) and [Gate](https://gate.minekube.com/) proxys through Velocity's modern forwarding.
+
+## Initialising modern forwarding
+
+It is as simple as adding the below between `MinecraftServer.init()` and `MinecraftServer#start`
+
+```java
+String secret = "My really secret secret that is not hard coded"
+VelocityProxy.enable(secret)
+```
+
+## Transferring between servers
+
+You need to install [Guva](https://github.com/google/guava) to transfer players. Once doing that, just add the below to send the appropriate packet
```

Comment:
You do not need guava to create this buffer. You can use the `NetworkBuffer` api:

Also should link to https://docs.papermc.io/paper/dev/plugin-messaging#what-did-we-just-do somewhere.

```
        p.sendPluginMessage("BungeeCord", NetworkBuffer.makeArray(buffer -> {
            buffer.write(NetworkBuffer.STRING, "Connect");
            buffer.write(NetworkBuffer.STRING, "lobby");
        }));
```